Errors when importing and/or drag and droping certain PDFs.

### JabRef version

Latest development branch build (please note build date below)

### Operating system

GNU / Linux

### Details on version and operating system

Ubuntu 20.04.5 - GNOME 3.36.5 -  JabRef 5.8

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

I have just started using JabRef and wanted to import all my local PDFs to create a bibliography file, but I encounter errors when importing certain PDF's : 

1. This happened with a specific PDF file . You can download it from [this google drive link](https://drive.google.com/file/d/1A-RD0vjLOC1ojsNgS0-A4iLMK6ugG8-L/view?usp=share_link)  
2. The errors arise when importing through the File->Import dialogue, as well as when drag and dropping. 
3. If multiple files were drag and dropped along with this one, the whole import operation stops once JabRef encounters the error.
4. The log file error is different between the approaches, you can see both in the appendix. 

I have tested this with JR 5.7 as well as the latest 5.8, but the error persists. It seems to be a problem with the xml parsing. 

### Appendix

...
<details>

<summary>Log File When Drag And Dropping</summary>

```
Problem parsing XMP schema. Continuing with other schemas.
java.io.IOException: org.apache.xmpbox.xml.XmpParsingException: Cannot find a definition for the namespace http://crossref.org/crossmark/1.0/
	at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilShared.parseXmpMetadata(Unknown Source)
	at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.getXmpMetadata(Unknown Source)
	at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.readXmp(Unknown Source)
	at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfXmpImporter.importDatabase(Unknown Source)
	at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfMergeMetadataImporter.importDatabase(Unknown Source)
	at org.jabref@5.8.239/org.jabref.logic.externalfiles.ExternalFilesContentImporter.importPDFContent(Unknown Source)
	at org.jabref@5.8.239/org.jabref.gui.externalfiles.ImportHandler$1.call(Unknown Source)
	at org.jabref@5.8.239/org.jabref.gui.externalfiles.ImportHandler$1.call(Unknown Source)
	at org.jabref@5.8.239/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
	at org.jabref.merged.module@5.8.239/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.apache.xmpbox.xml.XmpParsingException: Cannot find a definition for the namespace http://crossref.org/crossmark/1.0/
	at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.checkPropertyDefinition(Unknown Source)
	at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseChildrenAsProperties(Unknown Source)
	at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseDescriptionRoot(Unknown Source)
	at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parse(Unknown Source)
	... 16 more

```
</details>



<details>

<summary>Log File When Using Import Dialogue</summary>

```
Importing in unknown format...
Problem parsing XMP schema. Continuing with other schemas.
java.io.IOException: org.apache.xmpbox.xml.XmpParsingException: Cannot find a definition for the namespace http://prismstandard.org/namespaces/basic/2.0/
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilShared.parseXmpMetadata(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.getXmpMetadata(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.readXmp(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfXmpImporter.importDatabase(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfMergeMetadataImporter.importDatabase(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.lambda$importUnknownFormat$0(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.importer.ImportAction.doImport(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.importer.ImportAction.lambda$automatedImport$2(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
    at org.jabref.merged.module@5.8.239/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
    at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
    at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.apache.xmpbox.xml.XmpParsingException: Cannot find a definition for the namespace http://prismstandard.org/namespaces/basic/2.0/
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.checkPropertyDefinition(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseChildrenAsProperties(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseDescriptionRoot(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parse(Unknown Source)
    ... 19 more

Problem parsing XMP schema. Continuing with other schemas.
java.io.IOException: org.apache.xmpbox.xml.XmpParsingException: Cannot find a definition for the namespace http://crossref.org/crossmark/1.0/
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilShared.parseXmpMetadata(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.getXmpMetadata(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.readXmp(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfXmpImporter.importDatabase(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfMergeMetadataImporter.importDatabase(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.lambda$importUnknownFormat$0(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.importer.ImportAction.doImport(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.importer.ImportAction.lambda$automatedImport$2(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
    at org.jabref.merged.module@5.8.239/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
    at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
    at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.apache.xmpbox.xml.XmpParsingException: Cannot find a definition for the namespace http://crossref.org/crossmark/1.0/
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.checkPropertyDefinition(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseChildrenAsProperties(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseDescriptionRoot(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parse(Unknown Source)
    ... 19 more

Error importing
java.lang.IllegalArgumentException: prefix cannot be "null" when creating a QName
    at java.xml/javax.xml.namespace.QName.<init>(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomHelper.getQName(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseChildrenAsProperties(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parseDescriptionRoot(Unknown Source)
    at org.jabref.merged.module@5.8.239/org.apache.xmpbox.xml.DomXmpParser.parse(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilShared.parseXmpMetadata(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.getXmpMetadata(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.xmp.XmpUtilReader.readXmp(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfXmpImporter.importDatabase(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.fileformat.PdfMergeMetadataImporter.importDatabase(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.lambda$importUnknownFormat$0(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
    at org.jabref@5.8.239/org.jabref.logic.importer.ImportFormatReader.importUnknownFormat(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.importer.ImportAction.doImport(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.importer.ImportAction.lambda$automatedImport$2(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
    at org.jabref@5.8.239/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
    at org.jabref.merged.module@5.8.239/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
    at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
    at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    at java.base/java.lang.Thread.run(Unknown Source)

Import canceled

```
</details>

--- a/.github/workflows/check-links.yaml
+++ b/.github/workflows/check-links.yaml
@@ -4,6 +4,8 @@ on:
   push:
     paths:
       - '**.md'
+    paths-ignore:
+      - 'CHANGELOG.md'
   schedule:
     # Run on the first of each month at 9:00 AM (See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07)
     - cron: "0 9 1 * *"

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -57,6 +57,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 
 ### Fixed
 
+- We fixed an issue where importing from XMP would fail for certain PDFs [#9383](https://github.com/JabRef/jabref/issues/9383)
 - We fixed the readability of the file field in the dark dark theme [#9340](https://github.com/JabRef/jabref/issues/9340)
 - We fixed that sorting of entries in the maintable by special fields is updated immediately [#9334](https://github.com/JabRef/jabref/issues/9334)
 - We fixed the Cleanup entries dialog is partially visible [#9223](https://github.com/JabRef/jabref/issues/9223)

--- a/src/main/java/org/jabref/logic/xmp/XmpUtilShared.java
+++ b/src/main/java/org/jabref/logic/xmp/XmpUtilShared.java
@@ -9,7 +9,6 @@ import org.jabref.model.entry.BibEntry;
 
 import org.apache.xmpbox.XMPMetadata;
 import org.apache.xmpbox.xml.DomXmpParser;
-import org.apache.xmpbox.xml.XmpParsingException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
@@ -30,7 +29,8 @@ public class XmpUtilShared {
             DomXmpParser parser = new DomXmpParser();
             meta = parser.parse(is);
             return meta;
-        } catch (XmpParsingException e) {
+        } catch (Exception e) {
+            // bad style to catch Exception but as this is called in a loop we do not want to break here when any schema encounters an error
             throw new IOException(e);
         }
     }

